en/test_cards/MC_03 - Auto Manual mix

```diff
@@ -0,0 +1,37 @@
+Create and upload mission
+
+❏ Mission Criteria
```

Comment:
@hamishwillee we can also use real read-only checkboxes ere, it would look like this

```
- [ ] Mission Criteria
```

- [ ]  Mission Criteria

What do you think? @hamishwillee

Reply:
That doesn't work on gitbook - they will just render as `[ ] Stabilized`. It is a perfectly reasonable change, but I kind of like the boxes.